tools/cldr-apps/js/src/esm/cldrCla.mjs

```diff
@@ -0,0 +1,58 @@
+import * as cldrClient from "../esm/cldrClient.mjs";
+import * as cldrNotify from "../esm/cldrNotify.mjs";
+/**
+ * see ClaSignature.java
+ * @typedef {Object} ClaSignature
+ * @property {boolean} corporate true if a corporate signature
+ * @property {string} email
+ * @property {string} employer
+ * @property {string} name
+ * @property {boolean} unauthorized true if cla load failed
+ * @property {boolean} readonly true if cla may not be modified
+ * @property {boolean} signed true if signed, always true when returned from getCla()
```

Comment:
on the back end, it looks like "signed" is a Date rather than boolean?

Reply:
good catch, that should be the date signed.

Reply:
should that still be changed to a date, then?